test-result-summary-client/src/TabularView/TabularView.jsx

```diff
@@ -0,0 +1,703 @@
+                	// If benchmark aleady exists append to it
+                    if (newArray[currentEntry].benchmarkName == benchmarkName) {
+                        found = true;
+                        newArray[currentEntry].platforms[platform] = this.handleEntry(element, metric, type);
+                        break;
+                    }
+                }
+               // Create a new entry if benchmark name does not exist
+                if (!found) {
+                    entry = {};
+                    entry.platforms = {};
+                    entry.benchmarkName = element.aggregateInfo[0].benchmarkName + ',' + element.aggregateInfo[0].benchmarkVariant + ',' + element.aggregateInfo[0].metrics[metric].name;
+                    entry.platforms[platform] = this.handleEntry(element, metric, type);
+                    newArray.push(entry);	
+                }
+            }
+        }.bind(this));
+        if (type === 'test') {
+            this.setState({testData:newArray});
+        } else {
+            this.setState({baselineData:newArray});	
+        }
+    }
+
+    populateCompTable() {
+    	// Each entry is a combination of the data in testData and baselineData, same format two fields benchmarkName and platforms
+    	// Now platform entries contain information from both the testJdk and the baselineJdk
+        const newArray = [];
+
+        this.state.testData.forEach(function (elem) {
+            let entry = {};
+            let testEntry = elem;
+            let benchmark = testEntry.benchmarkName.split(",")[0];
+            let variant = testEntry.benchmarkName.split(",")[1];
+            let metric = testEntry.benchmarkName.split(",")[2];
+            entry.platforms = {};
+            entry.benchmarkName = testEntry.benchmarkName;
+
+            let higherBetter;
+            try {
+                if (benchmarkVariantsInfo[benchmark][variant][metric]["higherbetter"] === "t") {
+                    higherBetter = true;
+                } else {
+                    higherBetter = false;
+                }
+            } catch (higherBetterNotFoundError) {
+                higherBetter = true;
+            }
+
+            let element = this.state.baselineData.find( s => s.benchmarkName == elem.benchmarkName );
+            // Baseline data contains information for the benchmark, comparison possible
+            if (element != null) {
+            Object.keys(testEntry.platforms).forEach(function(platform) {
+            	// Baseline data contains same benchmark and same platform, compare values and store in comparison table
+                if (element.platforms.hasOwnProperty(platform)) {
+                    entry.platforms[platform] = {...testEntry.platforms[platform], ...element.platforms[platform]};	
+                    if (higherBetter) {
+                        entry.platforms[platform].compare = Number(testEntry.platforms[platform].testScore * 100 / element.platforms[platform].baselineScore).toFixed(2);
```

Comment:
If we have time, maybe we have some extra check to take care of high confidence interval.

If `CI 1 + CI 2 < Regression +0.7%`, then it's a confirmed regression. Else, we should show some symbol (i.e. warning sign or something) to show that confidence interval is high, and hence, we can't confirm the regression.